tests/integration/test_service.py

```diff
@@ -0,0 +1,211 @@
+from copy import copy
+from pathlib import Path
+from typing import Collection
+import random
+import string
+
+from httpx import AsyncClient
+import pytest
+import pytest_asyncio
+
+from adcm_aio_client.core.client import ADCMClient
+from adcm_aio_client.core.config import ConfigHistoryNode, ObjectConfig
+from adcm_aio_client.core.errors import ConflictError, MultipleObjectsReturnedError, ObjectDoesNotExistError
+from adcm_aio_client.core.filters import Filter
+from adcm_aio_client.core.objects._imports import Imports
+from adcm_aio_client.core.objects.cm import Cluster, License, Service
+from tests.integration.bundle import pack_bundle
+from tests.integration.yaml import create_yaml
+
+pytestmark = [pytest.mark.asyncio]
+
+
+def prepare_bundle_data() -> list[dict]:
+    config = [
+        {
+            "name": "string_field",
+            "type": "string",
+            "default": "string_field value",
+        }
+    ]
+
+    service = {
+        "type": "service",
+        "name": "Generated service",
+        "version": 1.0,
+        "config": config,
+    }
+
+    service_manual_add = copy(service)
+    service_manual_add.update({"name": "Manual add", "license": "./service_license.txt"})
+
+    fifty_one_services = []
+    for i in range(51):
+        service = copy(service)
+        service.update({"name": f"Generated service {i + 1}"})
+        fifty_one_services.append(service)
+
+    return [
+        {
+            "type": "cluster",
+            "name": "Generated cluster",
+            "version": 1,
+        },
+        *fifty_one_services,
+        service_manual_add,
+    ]
+
+
+def assert_services_collection(services: Collection[Service], expected_amount: int) -> None:
+    assert all(isinstance(cluster, Service) for cluster in services)
+    assert len({service.id for service in services}) == expected_amount
+    assert len({id(service) for service in services}) == expected_amount
+
+
+@pytest_asyncio.fixture()
+async def cluster_52(adcm_client: ADCMClient, tmp_path: Path) -> Cluster:
+    """
+    Cluster with 52 services, one not added
+    """
+
+    bundle_folder = tmp_path / "".join(random.sample(string.ascii_letters, k=6)).lower()
+    config_yaml_path = bundle_folder / "config.yaml"
+    create_yaml(data=prepare_bundle_data(), path=config_yaml_path)
+
+    with (bundle_folder / "service_license.txt").open("wt") as service_license_file:
+        service_license_file.write("By using this test bundle, you agreeing to write tests well\n")
```

Comment:
you can use `write_text` directly on Path object

Reply:
fixed